components/Button.js

```diff
@@ -0,0 +1,11 @@
+export default function Button({
+  children, link, backgroundstyle, textstyle,
+}) {
+  return (
+    <div className={backgroundstyle}>
+      <a href={link}>
+        <span className={textstyle}>{children}</span>
+      </a>
+    </div>
```

Comment:
Use button tag instead of div for defining a buttonis better for accesibility an semantics. Also the anchor is unnecesary, you can create the button and set a click event to push the url link.